mlflow/utils/autologging_utils/safety.py

```diff
@@ -517,10 +519,12 @@ def call_original(*og_args, **og_kwargs):
                         e,
                     )
 
+                    import traceback
                     _logger.warning(
-                        "Encountered unexpected error during %s autologging: %s",
+                        "Encountered unexpected error during %s autologging: %s, traceback: %s",
                         autologging_integration,
                         e,
+                        traceback.format_stack()
```

Comment:
Is this a temporary change to make it easier to debug errors?

Reply:
yes, will revert later